packages/node-cli/src/core/authentication/AuthenticationService.js

```diff
@@ -53,4 +64,19 @@ module.exports = class AuthenticationService {
 			}
 		}
 	}
+
+	async getAuthIds(sdkExecutor) {
+		const getAuthListContext = new SdkExecutionContext({
+			command: COMMANDS.MANAGEAUTH,
+			flags: [FLAGS.LIST],
+		});
+		const existingAuthIDsResponse = await executeWithSpinner({
+			action: sdkExecutor.execute(getAuthListContext),
+			message: NodeTranslationService.getMessage(COMMAND_SETUPACCOUNT.MESSAGES.GETTING_AVAILABLE_AUTHIDS),
+		});
+		if (existingAuthIDsResponse.status === SdkOperationResultUtils.STATUS.ERROR) {
+			throw SdkOperationResultUtils.getResultMessage(existingAuthIDsResponse);
```

Comment:
In general I think is will be better to use the SdkOperationResultUtils.collectErrorMessages, it can log more information if it's present.